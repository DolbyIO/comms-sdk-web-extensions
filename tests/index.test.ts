import './__mocks__/MediaStream.mock';
import './__mocks__/navigator.mock';
import VoxeetSDKExt from '../src/index';

describe('index test suite', () => {
    test('index should match the snapshot', () => {
        expect(VoxeetSDKExt).toMatchSnapshot();
    });
});
