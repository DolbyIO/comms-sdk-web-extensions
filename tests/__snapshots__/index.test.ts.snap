// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index test suite index should match the snapshot 1`] = `
Object {
  "conference": Object {
    "attachMediaStreamToHTMLVideoElement": [Function],
    "switchToListener": [Function],
    "switchToUser": [Function],
  },
  "spatialAudio": Object {
    "createPrivateZone": [Function],
    "deletePrivateZone": [Function],
    "privateZones": Map {},
    "setSpatialEnvironment": [Function],
    "setSpatialPosition": [Function],
    "updatePrivateZone": [Function],
  },
}
`;
